test/unit/request-logger/index.js

```diff
@@ -66,3 +66,20 @@ test('should return body content', t => {
   const { body } = reqLogger(req)
   t.deepEqual(body, expectedBody)
 })
+
+test('should return the correctly url', t => {
```

Comment:
should return req.originalUrl's value over req.url\'s

ou se achar como está melhor  só mudar de "correctly url" para " correct url" 

O que preferir :)